src/Sulu/Bundle/WebsiteBundle/DependencyInjection/Configuration.php

```diff
@@ -36,6 +36,12 @@ public function getConfigTreeBuilder()
                     ->scalarNode('analytics_key')->defaultValue('UA-SULU-PREVIEW-KEY')->end()
                 ->end()
             ->end()
+            ->arrayNode('analytics')
+                ->addDefaultsIfNotSet()
+                ->children()
+                    ->booleanNode('enabled')->defaultTrue()->end()
+                ->end()
+            ->end()
```

Comment:
Why don't you use the `canBeDisabled` call as described in http://symfony.com/doc/current/components/config/definition.html#optional-sections ?